### PR TITLE
Switch to core Keras GroupNormalization

### PR DESCRIPTION
**Pull request by fchollet**
The layer being swapped out is numerically identical. I also tested the generation quality by hand.

The layer is in 2.11, the latest version, and soon to be second-latest (maybe 2.10 as well? Haven't checked).

### CODE REVIEW

**Comment by LukeWood** [comment]
> The layer being swapped out is numerically identical. I also tested the generation quality by hand.
> 
> The layer is in 2.11, the latest version, and soon to be second-latest (maybe 2.10 as well? Haven't checked).

Thanks for remembering to update this Francois.  Can you update https://github.com/keras-team/keras-cv/blob/master/keras_cv/version_check.py#L21 with this?

**Comment by fchollet** [comment]
Done. I checked that the first version that has the layer is in fact 2.11, not 2.10.

**Comment by LukeWood** [comment]
> Done. I checked that the first version that has the layer is in fact 2.11, not 2.10.

two tests to update too - sorry about that: https://github.com/keras-team/keras-cv/blob/master/keras_cv/version_check_test.py#L40

**Review by LukeWood — CHANGES_REQUESTED** [review]
Test failure - here's a link:  https://github.com/keras-team/keras-cv/blob/master/keras_cv/version_check_test.py#L40

**Review by LukeWood — APPROVED** [review]

**Comment by ianstenbit** [comment]
Thanks for this! Sorry I forgot to update this after upstreaming GroupNorm to core Keras.
@LukeWood did you verify that this doesn't break fine-tuning flows (since those aren't unit tested currently)?

**Comment by LukeWood** [comment]
> Thanks for this! Sorry I forgot to update this after upstreaming GroupNorm to core Keras. @LukeWood did you verify that this doesn't break fine-tuning flows (since those aren't unit tested currently)?

@fchollet checked numerical correctness on both ends - group normalization and SD.  I'm fairly confident it didn't break them but will run on Colab.
